docs/source/general/abstract_models.rst

```diff
@@ -127,43 +153,44 @@ and make the following changes:
-                ('secret', models.CharField(help_text='Shared Secret', max_length=60, verbose_name='secret')),
-                ('ports', models.IntegerField(blank=True, null=True, verbose_name='ports')),
-                ('community', models.CharField(blank=True, max_length=50, null=True, verbose_name='community')),
-                ('description', models.CharField(max_length=200, null=True, verbose_name='description')),
-                ('server', models.CharField(max_length=64, null=True, verbose_name='server')),
-            ],
-            options={
-                'db_table': 'nas',
-                'swappable': swapper.swappable_setting('django_freeradius', 'Nas'),
-                'verbose_name': 'nas',
-                'abstract': False,
-                'verbose_name_plural': 'nas',
-            },
-        ),
+    #django_freeradius/migrations
+
+    import swapper
+
+
+    class Migration(migrations.Migration):
+
+        initial = True
+
+        dependencies = [
+            swapper.dependency('django_freeradius', 'RadiusReply'),
+            swapper.dependency('django_freeradius', 'RadiusCheck'),
+        ]
+
+        operations = [
+            migrations.CreateModel(
+                name='Nas',
+                fields=[
+                    ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                    ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, editable=False, verbose_name='created')),
+                    ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, editable=False, verbose_name='modified')),
+                    ('nas_name', models.CharField(db_column='nasname', db_index=True, help_text='NAS Name (or IP address)', max_length=128, unique=True, verbose_name='nas name')),
+                    ('short_name', models.CharField(db_column='shortname', max_length=32, verbose_name='short name')),
+                    ('type', models.CharField(max_length=30, verbose_name='type')),
+                    ('secret', models.CharField(help_text='Shared Secret', max_length=60, verbose_name='secret')),
+                    ('ports', models.IntegerField(blank=True, null=True, verbose_name='ports')),
+                    ('community', models.CharField(blank=True, max_length=50, null=True, verbose_name='community')),
+                    ('description', models.CharField(max_length=200, null=True, verbose_name='description')),
+                    ('server', models.CharField(max_length=64, null=True, verbose_name='server')),
+                ],
+                options={
+                    'db_table': 'nas',
+                    'swappable': swapper.swappable_setting('django_freeradius', 'Nas'),
+                    'verbose_name': 'nas',
+                    'abstract': False,
+                    'verbose_name_plural': 'nas',
+                },
+            ),
```

Comment:
let's not duplicate the migration code, remove this and simply explain that a migration needs to be created if is necessary.